Resources/Private/Partials/UserList.html

```diff
@@ -24,9 +24,7 @@
 				</a>
 				<div class="beuser-fastswitch__useritem-actions">
 					<f:render section="Info" arguments="{user: user}"/>
-					<f:if condition="!{user.isDisabled}">
-						<f:render section="Switch" arguments="{user: user}" />
-					</f:if>
```

Comment:
So you want to change it to make it possible switching to a deactivated user on purpose? @tdeuling